MdeModulePkg/Library/DeviceStateLib/DeviceStateLib.c

```diff
@@ -1,4 +1,4 @@
-/** @file
+/** @file DeviceStateLib.c
 Functions used to support Getting and Setting Device States.

```

Comment:
Since this talks about `PcdDeviceStateBitmask`, you might also want to describe how this library instance uses `PcdInsecureDeviceState` (i.e. fixed PCD).